ui/src/adapters/parsers/forms.ts

```diff
@@ -143,33 +206,13 @@ const parseIssueCredentialFormDataAttributes = (
     },
     { data: [], success: true }
   );
-};
+}
 
-export const parseLinkExpirationDate = StrictSchema<{
-  linkExpirationDate: dayjs.Dayjs | null;
-}>()(
-  z.object({
-    linkExpirationDate: dayjsInstance.nullable(),
-  })
-);
+// Exports
```

Comment:
I'd remove this comment, there are exported elements above.

Reply:
Thanks, I'll revise.